docker/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: "2"
+services: 
+  rocketchat-tests-based-on-splinter:
+    image: cusdeb/rocketchat-tests-based-on-splinter
+    command: "-s all"
+    links:
+      - "rocketchat:rocketchat"
+    environment: 
+      - USERNAME=admin
+      - PASSWORD=qwerty12+
+      - ADDR=rocketchat
+      - PORT=8006
+      - HOST=http://rocketchat:8006
+      - BOT_NAME=meeseeks
+      - BRANCH=wizard-integration
+  rocketchat:
+    image: cusdeb/rocketchat:0.70-amd64
+    links: 
+      - "mongodb:mongodb"
+    environment:
+    - PORT=8006
+    - MONGO_DATABASE=rocketchat
+    - MONGO_HOST=mongodb:27017
+    - ROOT_URL=http://127.0.0.1:8006
+  mongodb:
+    image: mongo:3.2-jessie
+  hubot-rocketchat:
+    image: cusdeb/hubot-rocketchat:2.0-amd64
+    links: 
+      - "rocketchat:rocketchat"
+      - "redis:redis"
+    environment: 
+      # Rocket.Chat
+      - AUTH_ATTEMPTS=240
+      - EXTERNAL_SCRIPTS=git:tolstoyevsky/hubot-viva-las-vegas,git:tolstoyevsky/hubot-vote-or-die,git:tolstoyevsky/hubot-pugme
+      - ROCKETCHAT_PASSSWORD=pass
+      - ROCKETCHAT_URL=http://rocketchat:8006
+      - ROCKETCHAT_USER=meeseeks
+      # Redis
+      - REDIS_URL=redis://redis:16379
```

Comment:
I suggest we will remove `REDIS_CONF_port` and use only `REDIS_URL=redis://redis` without port specifying.